app/components/dropdown_menu_component.rb

```diff
@@ -3,12 +3,13 @@
 class DropdownMenuComponent < ViewComponent::Base
   renders_one :icon
 
-  def initialize(menu_title:, icon_name: nil, hide_label: false, render_check: true, klass: nil)
+  def initialize(menu_title:, icon_name: nil, hide_label: false, render_check: true, klass: nil, button_klass: nil)
     @menu_title = menu_title
     @render_check = render_check
     @hide_label = hide_label
     @icon_name = icon_name
     @class = klass
+    @button_class = button_klass
```

Comment:
`button_class` makes sense. Lets take that 1 step further and lets make `klass` `container_class`